graf2d/gpad/src/TRatioPlot.cxx

```diff
@@ -714,7 +744,7 @@ TAxis* TRatioPlot::GetLowerRefXaxis()
 /// rp->GetLowerRefGraph()->GetYaxis();
 /// ~~~
 
-TAxis* TRatioPlot::GetLowerRefYaxis()
+TAxis* TRatioPlot::GetLowerRefYaxis() const
```

Comment:
@karies But can I write doxygen comments if they're inline? That's why I put them out-of-line